packages/apps/human-app/frontend/src/pages/worker/jobs/jobs.page.tsx

```diff
@@ -48,14 +53,20 @@ export function JobsPage() {
     }
   };
 
+  useEffect(() => {
+    setGrayBackground();
+  }, [setGrayBackground]);
+
+  useEffect(() => {
```

Comment:
Seems redundant

Reply:
Yes, we can reduce it to just 
```
if (error) {
      void onError(error);
}
````
and it will work the same, but (IMO) looks a bit worse code-wise :sweat_smile:.
Do You want to me to remove the `useEffect`? It's basically a preferences matter :smile: